Lib9c/Action/RequestPledge.cs

```diff
@@ -0,0 +1,58 @@
+using Bencodex.Types;
+using Lib9c;
+using Libplanet;
+using Libplanet.Action;
+using Libplanet.State;
+using Nekoyume.Model.State;
+
+namespace Nekoyume.Action
+{
+    [ActionType(TypeIdentifier)]
+    public class RequestPledge : ActionBase
+    {
+        public const string TypeIdentifier = "request_pledge";
+
+        public RequestPledge()
+        {
+        }
+
+        // Value from tx per block policy.
+        // https://github.com/planetarium/lib9c/blob/b6c1e85abc0b93347dae8e1a12aaefd767b27632/Lib9c.Policy/Policy/MaxTransactionsPerSignerPerBlockPolicy.cs#L29
+        public const int RefillMead = 4;
+        public Address AgentAddress;
+        public int Mead;
```

Comment:
This field doesn't seem to be used.

Reply:
this field using configure refill mead value. i'll rename `RefillMead`